circuits/core/manager.py

```diff
@@ -894,6 +894,7 @@ def processTask(self, event, task, parent=None):  # noqa
                 self._eventDone(event)
         except KeyboardInterrupt:
             self.stop()
+            raise
```

Comment:
Can you please explain this?

Reply:
I reverted it, it was not necessary.

Reply:
I looked through it a bit. I think I added it because it is bad style to swallow keyboard interupts. This way the user cannot hit `ctrl+c` to interrupt the program (unless `self.stop` does this).

Reply:
Yes, it should be addressed in a separate issue.